rest/route/select_test.go

```diff
@@ -0,0 +1,107 @@
+package route
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSelectTestsHandler(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	j := []byte(`{
```

Comment:
Should we wrap this test unit (and the invalid test cases below) in their own `t.Run("...`? It would improve the signaling and make it easier to decipher what's testing where.

Reply:
I think these tests are simple enough, and likely enough to change, that adding additional text is sufficient.